app/src/main/java/com/andrewKarachun0304/tmshomeworkandroid/hw9/HomeWork9Activity.kt

```diff
@@ -0,0 +1,46 @@
+package com.andrewKarachun0304.tmshomeworkandroid.hw9
+
+import android.content.Intent
+import android.os.Build
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.os.CountDownTimer
+import androidx.annotation.RequiresApi
+import com.andrewKarachun0304.tmshomeworkandroid.R
+import kotlinx.android.synthetic.main.activity_home_work9.*
+import java.util.*
+
+class HomeWork9Activity : AppCompatActivity() {
+    private var timer: CountDownTimer? = null
+
+    @RequiresApi(Build.VERSION_CODES.O)
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_home_work9)
+
+        val serviceIntent = Intent(this@HomeWork9Activity, TimerService::class.java)
+
+        start_btn1.setOnClickListener {
+            val second = time_picker.minute * 60
+            timer = object : CountDownTimer(second * 1000L, 1000) {
+                override fun onTick(p0: Long) {
+                    val sec = p0 / 1000 % 60
+                    val min = (p0 / 1000 % 3600) / 60
```

Comment:
What for you need %3600.If remains m:s we have (m*60+seconds)*1000.So if we divide on 1000 and than divide on 60 we will get minutes

Reply:
i want use hours, but i forgot about it, fixed